orchestra/static/orchestra/todos/todo-list.directive.es6.js

```diff
@@ -101,6 +101,16 @@ export default function todoList (orchestraApi) {
         $scope.$apply()
       }
 
+      todoList.transformToTree = (todos) => {
+        var nodes = {}
+        return todos.filter(function (obj) {
+          nodes[obj.id] = defaults(obj, nodes[obj.id], { items: [] })
+          obj.parent_todo && (nodes[obj.parent_todo] = (nodes[obj.parent_todo] || { items: [] }))['items'].push(obj)
+
+          return !obj.parent_todo
```

Comment:
Sanity check: return [] when `todos` is empty is the desired behavior?

Reply:
if todos is [], transformToTree returns []